os/board/rtl8730e/src/rtl8730e_mipi_lcdc.c

```diff
@@ -171,11 +173,9 @@ static void rtl8730e_lcd_put_area(u8 *lcd_img_buffer, u32 x_start, u32 y_start,
 #endif
 	LCDC_LayerConfig(pLCDC, LCD_LAYER, &lcdc_init_struct.layerx[LCD_LAYER]);
 	DCache_CleanInvalidate((u32)lcd_img_buffer, LCDC_IMG_BUF_SIZE);
-	lcdc_nextframe = 0;
+	sem_wait(&g_next_frame_block);
+	lcdc_nextframe = 1;
```

Comment:
this flag is checked on thread and IRQ, So I thick, It is needed to protect critical section using irqsave()